src/App.tsx

```diff
@@ -1,3 +1,18 @@
-import React from 'react';
+// this comment tells babel to convert jsx to calls to a function called jsx instead of React.createElement
```

Comment:
I'm surprised you need this if you're using `@emotion/babel-preset-css-prop`. Is this needed for compilation, or for integration with other projects?

Reply:
Storybook wouldn't work without this. I've no idea why not, but it wasn't happy unless I added this in. I'd love to get it working without it because I've not seen any other Guardian projects need it

Reply:
Getting Babel and Storybook to play together has been a bit weird. Storybook respects `.babelrc` but I'm not sure about `.babelrc.js`. You might need to create a [`.storybook/.babelrc`](https://storybook.js.org/docs/react/configure/babel#custom-config-file) file. For legacy reasons I can't quite remember, Source creates a [custom Webpack config](https://github.com/guardian/source/blob/da9b2181a2ce0e8a8b3eeccc3ddd716d1a31e0f3/.storybook/webpack.config.js#L11) 😱

Reply:
Swapping to `.babelrc` from `.babelrc.js` fixed this.